src/main/cli/cli.c

```diff
@@ -939,10 +939,9 @@ static void printAux(uint8_t dumpMask, const modeActivationCondition_t *modeActi
     // print out aux channel settings
     for (uint32_t i = 0; i < MAX_MODE_ACTIVATION_CONDITION_COUNT; i++) {
         const modeActivationCondition_t *mac = &modeActivationConditions[i];
-        bool equalsDefault = false;
+        const bool equalsDefault = !isModeActivationConditionConfigured(mac);
```

Comment:
Yes, I like this better. The recreation of the `emptyMac` bothered me as well but at the same time it's an initialization phase rather than in a critical loop so it wasn't that bad. But allocating once and passing as a parameter is a better idea. I'll fix up.

Side question: getting a lot of commits on this. Want me to squash or keep the individual commits?

Reply:
Yes, please squash.

Reply:
Updated `isModeActivationConditionConfigured()` logic, rebased, and squashed.